staging/src/k8s.io/kubectl/pkg/cmd/get/get_test.go

```diff
@@ -168,7 +168,7 @@ func TestGetObjectsWithOpenAPIOutputFormatPresent(t *testing.T) {
 	defer tf.Cleanup()
 	codec := scheme.Codecs.LegacyCodec(scheme.Scheme.PrioritizedVersionsAllGroups()...)
 
-	// overide the openAPISchema function to return custom output
+	// override the openAPISchema function to return custom output
```

Comment:
The same problem exists in other files，It would be better if ould be modified together, thks

./cmd/kubeadm/app/phases/kubelet/flags.go
// - "hostname-overide" flag in NodeRegistrationOptions.KubeletExtraArgs

Reply:
@heqg Thanks notification!
Fixed it.